app/modules/ia_config_reader.py

```diff
@@ -0,0 +1,109 @@
+# -*- coding: utf-8 -*-
+
+import logging
+import json
+import os.path as path
+
+log = logging.getLogger(__name__)
+
+
+def load_config_to_dict(fname):
+    app_home = ''
```

Comment:
what's the benefit of app_home being a variable, seems pointless unless I'm missing something.

Reply:
I like vars like that and single-use funcs for readability, but am not too attached. Removing such cases since we seem to prefer minimal code here.